interview-questions.md

```diff
@@ -8,62 +8,61 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-  Your answer:
+  Your answer: A variable that holds data
 
-  Researched answer:
+  Researched answer: Values that are passed into a function.
```

Comment:
Kinda. The questions is asking for the difference between the two things which is that one is the placeholder in the function and one is the actual value that is passed in.